### PR TITLE
Naming things and cat invalidation amirite

### PR DESCRIPTION
**Pull request by hollerith**

### CODE REVIEW

File: trust_and_safety_models/abusive/abusive_model.py [hunk 1]
```diff
@@ -15,7 +15,10 @@
 import utils
 
 cat_names = [
-...
+    'Tiddles',
+    'MrMeowGi',
+    'FidelCatsro',
+    'Psppspsps'
```
Comment (randiaz95) [inline trust_and_safety_models/abusive/abusive_model.py:21]:
```suggestion
    'Supreme Leader Meow'
```

Comment (crissilvaeng) [inline trust_and_safety_models/abusive/abusive_model.py:21]: Pedro Pspspscal is a beloved and popular cat in the geek community that has gained significant attention due to his unique and charming personality. By adding Pedro Pspspscal to the `cat_names` array, you aim to achieve the following:

**Inclusivity:** Including _Pedro Pspspscal_ in the list of cat names ensures that you represent a diverse range of feline personalities and acknowledge their impact on the community.

**Engagement:** _Pedro Pspspscal_'s presence in the array will likely increase user engagement. His popularity will draw more users to interact with the platform, generating positive sentiment and potentially increasing user retention.

**Relevance:** Adding trending and popular cat names like _Pedro Pspspscal_ keeps the platform up-to-date and relevant, ensuring users find value and enjoyment in the content provided.

```suggestion
    'Psppspsps',
    'Pedro Pspspscal'
```

Comment (hollerith) [reply trust_and_safety_models/abusive/abusive_model.py:21]: Chairman Meow WAS RIGHT THERE

**Comment by hollerith** [comment]
WHY TWTITTR CLOSE THIS POPULAR PR

**Pull request closed without merging**